content/tutorial-deep-learning-on-mnist.ipynb

```diff
@@ -0,0 +1,1067 @@
+{
```

Comment:
<p>I think it might be better to leave all credits at the bottom of the document.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
With this one, I moved it to the bottom. That repo helped find an alternative to the `keras` shortcut for dataset downloading, but since then you've revised that part of the code quite a bit 👍